js/donations-visualiser.js

```diff
@@ -190,6 +190,9 @@ var data_request = d3.json("data/all_data.json")
         $("#loading-modal").modal('hide');
         processData(data);
 
+        // zoom out a ways initially
+        zoomTo(0.5).event(container);
```

Comment:
this seems to give a slightly weird visual effect whereby the site loads, zooms, and then zooms to fit. not a big deal, but maybe better without this?